action.yml

```diff
@@ -21,6 +21,14 @@ inputs:
   sonar-api-url:
     description: Base URL of the SonarCloud API.
     default: https://sonarcloud.io/api
+  defectdojo-token:
+    description: Token for authenticating requests to DefectDojo.
+    required: false
+  defectdojo-api-url:
+    description: Base URL of the DefectDojo API.
```

Comment:
I think this needs to be `required: false` as well,  or else it will end up being required for sonar, semgrep, etc.

Reply:
@drdavella which would be the default url?

Reply:
@carlosu7 I don't think we can have a default here. The problem is that if we make this `required: true` then it will be required when the tool is `semgrep` or `sonar` which doesn't make sense.

Instead we might need to add some validation in the code like:
```python
# Please excuse my python pseudocode
if tool == "defectdojo" and not defectdojo_api_url:
    raise Error("Require API URL for tool DefectDojo")
```

Reply:
@drdavella done